test/test.cpp

```diff
@@ -163,3 +173,25 @@ TEST(Clip, Points) {
     ASSERT_EQ(expected1, clipped1);
     ASSERT_EQ(expected2, clipped2);
 }
+
+TEST(GetTile, USStates) {
+    const auto geojson = mapbox::geojson::parse(loadFile("test/fixtures/us-states.json"));
+    GeoJSONVT index{ geojson.get<mapbox::geojson::feature_collection>() };
+
+    const auto features = index.getTile(7, 37, 48).features;
+    const auto expected = parseJSONTile(loadFile("test/fixtures/us-states-z7-37-48.json"));
+    ASSERT_EQ(features == expected, true);
+
+    const auto square = parseJSONTile(loadFile("test/fixtures/us-states-square.json"));
+    ASSERT_EQ(square == index.getTile(9, 148, 192).features, true);  // clipped square
+    ASSERT_EQ(square == index.getTile(11, 592, 768).features, true); // clipped square
+
+    ASSERT_EQ(empty_tile == index.getTile(11, 800, 400), true);  // non-existing tile
+    ASSERT_EQ(&empty_tile == &index.getTile(11, 800, 400), true); // non-existing tile
+
+    // This test does not make sense in C++, since the parameters are cast to integers anyway.
+    // ASSERT_EQ(isEmpty(index.getTile(-5, 123.25, 400.25)), true); // invalid tile
+
+    ASSERT_EQ(29, index.total);
```

Comment:
I think the algorithm changed a bit (e.g. it now generates empty tiles in some cases, making total bigger) so if the results look sane, just updated expected values. `getTotal()` is equivalent.